src/Umbraco.Cms.Api.Management/Controllers/Webhook/AllWebhookController.cs

```diff
@@ -0,0 +1,38 @@
+using Asp.Versioning;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Api.Common.ViewModels.Pagination;
+using Umbraco.Cms.Api.Management.ViewModels.Webhook;
+using Umbraco.Cms.Core.Mapping;
+using Umbraco.Cms.Core.Models;
+using Umbraco.Cms.Core.Services;
+
+namespace Umbraco.Cms.Api.Management.Controllers.Webhook;
+
+[ApiVersion("1.0")]
+public class AllWebhookController : WebhookControllerBase
+{
+    private readonly IWebhookService _webhookService;
+    private readonly IUmbracoMapper _umbracoMapper;
+
+    public AllWebhookController(IWebhookService webhookService, IUmbracoMapper umbracoMapper)
+    {
+        _webhookService = webhookService;
+        _umbracoMapper = umbracoMapper;
+    }
+
+    [HttpGet]
+    [MapToApiVersion("1.0")]
+    [ProducesResponseType(typeof(PagedViewModel<WebhookResponseModel>), StatusCodes.Status200OK)]
+    public async Task<ActionResult<PagedViewModel<WebhookResponseModel>>> All(int skip = 0, int take = 100)
+    {
+        IWebhook[] webhooks = (await _webhookService.GetAllAsync(skip, take)).Items.ToArray();
+        var viewModel = new PagedViewModel<WebhookResponseModel>
+        {
+            Total = webhooks.Length,
+            Items = _umbracoMapper.MapEnumerable<IWebhook, WebhookResponseModel>(webhooks)
+        };
+
+        return await Task.FromResult(Ok(viewModel));
```

Comment:
For redirects the controller is named `GetAllRedirectUrlManagementController` and the method `GetAll()` instead of `All()` here and for languages.

Would be great to streamline that if possible.

Reply:
`All` is the correct name, as Get is implied by the `HttpGet`.
Yes it should just return `Ok(viewModel)` as we're making an async call with `GetAllAsync` 👍

Reply:
I updated that and also updated it a bit for `All` in language controller.